base/show.jl

```diff
@@ -1919,7 +1919,7 @@ function show_unquoted(io::IO, ex::Expr, indent::Int, prec::Int, quote_level::In
             na = length(func_args)
             if (na == 2 || (na > 2 && isa(func, Symbol) && func in (:+, :++, :*)) || (na == 3 && func === :(:))) &&
                     all(a -> !isa(a, Expr) || a.head !== :..., func_args)
-                sep = func === :(:) ? "$func" : " $func "
+                sep = func === :(:) ? "$func" : " " * repr(func) * " "   # if func::Any, avoid string interpolation (invalidation)
```

Comment:
We have `func::Symbol` asserted just above

Reply:
Only if `na > 2`?

Reply:
Ah, right, the first argument call. `$` and `repr` do not produce the same result for Symbol though

Reply:
Yeah, that was definitely a mistake.